problems/27.Check If Two String Arrays are Equivalent.gs

```diff
@@ -0,0 +1,15 @@
+function arrayStringsAreEqual(word1: string[], word2: string[]): boolean {
+    let str1: string = strings(word1)
+    let str2: string = strings(word2)
+    return str1 == str2
+};
+function strings(words: string[]): string {
+    let str: string = words.toString()
```

Comment:
There is a lot better way to do this, this way is really slow.
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/join